src/windows/system.rs

```diff
@@ -158,8 +158,8 @@ impl SystemExt for System {
             let mut mem_info: MEMORYSTATUSEX = zeroed();
             mem_info.dwLength = size_of::<MEMORYSTATUSEX>() as u32;
             GlobalMemoryStatusEx(&mut mem_info);
-            self.mem_total = auto_cast!(mem_info.ullTotalPhys, u64) / 1_000;
-            self.mem_available = auto_cast!(mem_info.ullAvailPhys, u64) / 1_000;
+            self.mem_total = auto_cast!(mem_info.ullTotalPhys, u64) / 1_024;
+            self.mem_available = auto_cast!(mem_info.ullAvailPhys, u64) / 1_024;
```

Comment:
And based on this page: https://docs.microsoft.com/en-us/windows/win32/api/sysinfoapi/ns-sysinfoapi-memorystatusex, it's all in bytes so this conversion is unneeded as well.